fix: restrict secrets tab usage to admin users

VikaCep opened the pull request:
From https://raintank-corp.slack.com/archives/C07D9QWFTQ8/p1753822656278269?thread_ts=1753720623.506399&cid=C07D9QWFTQ8

As a first stage, we need to restrict the content of the Secrets tab to just Admins.

**Viewer/Editor**
<img width="1483" height="602" alt="image" src="https://github.com/user-attachments/assets/7990a3fa-0c54-4825-80d3-e8626aa6de6e" />

**Admin**
<img width="1209" height="709" alt="image" src="https://github.com/user-attachments/assets/04e231e2-feca-4e7b-94ec-f0992bd03fc6" />

I reused the `ContactAdminAlert` but I'm open to changing it to other UI if preferred.

At a later stage, we'll focus on adding more granular permissions, but it's out of the scope of this PR.

ckbedwell reviewed (approved):
In the context of the linked thread: LGTM.

_(Otherwise, booooooo. This change sucks 😅  -- let's get a ticket to get those RBAC permissions in sooner rather than later )_

VikaCep commented:
> (Otherwise, booooooo. This change sucks 😅 -- let's get a ticket to get those RBAC permissions in sooner rather than later )

We could easily consume the fixed roles described in the thread and apply them in the UI:

```
fixed:secret.securevalues:creator
fixed:secret.securevalues:reader
fixed:secret.securevalues:updater
fixed:secret.securevalues:deleter
```

But this would still allow users to access them via the API, so we need to sync with the backend to make sure how to address that.

VikaCep commented:
Here's the [PR](https://github.com/grafana/synthetic-monitoring-app/pull/1218) with RBAC permissions applied on the UI side
